cmd/pitr/recoverer/recoverer.go

```diff
@@ -272,6 +294,47 @@ func getLastBackupGTID(infoObj io.Reader) (string, error) {
 	return string(set), nil
 }
 
+func getDecompressedContent(infoObj io.Reader, format string) ([]byte, error) {
+	tmpDir := os.TempDir()
+	fileName := "backup_info." + format
+	tmpFile, err := os.Create(tmpDir + "/" + fileName)
```

Comment:
why we need temp file? we can write directly to STDIN on executed command

Reply:
done https://github.com/percona/percona-xtradb-cluster-operator/pull/743/commits/acb9a424f577f3a06fdf388b2fdce0775c2157a9